docs/en/stack/ml/df-analytics/ml-lang-ident.asciidoc

```diff
@@ -118,12 +118,14 @@ POST _ingest/pipeline/_simulate
 ----------------------------------
 //NOTCONSOLE
 
-<1> The ID of the {lang-ident} trained model.
-<2> Indicates that only the top five languages (that is to say, the ones with the highest probability) are reported.
-In this example, 5 classes (in this case, languages) with the 
-highest probability will be reported.
-<3> The source object that contains the text to identify.
+<1> ID of the {lang-ident} trained model.
+<2> Specifies the number of languages to report by descending order of 
+probability.
+<3> Source object containing the text to identify.
```

Comment:
> Source object containing...

Likewise, I'm not sure this is a clearer wording.

Reply:
As above.